datasets.py

```diff
@@ -919,3 +920,110 @@ def from_path(self, file_path):
         anns = [image_anns, gender_info, [country], file_path, scene_group]
 
         return image, anns
+
```

Comment:
we don't have this for other datasets (and tbh we probably should), but can you include as a comment here where one could find this dataset and how to download it

Reply:
Done!